content/webapp/pages/pages/[pageId].tsx

```diff
@@ -108,7 +108,13 @@ export const getServerSideProps: GetServerSideProps<
   const { pageId } = context.query;
   const siteSection = toMaybeString(context.params?.siteSection);
 
-  if (!looksLikePrismicId(pageId)) {
+  // We don't allow e.g. /visit-us/visit-us as it's a landing page
+  // Should only display on /visit-us
+  const isLandingPageRenderingAsSubPage =
+    pageId === siteSection &&
+    context.resolvedUrl.indexOf(`/${siteSection}/${pageId}`) === 0;
```

Comment:
A little bit verbose to my taste, will take ideas on cleverer ways of checking this.